<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>listcreator</class>
 <widget class="QDialog" name="listcreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Word List Creator - Word Search Creator</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>Open File...</string>
       </property>
       <property name="icon">
        <iconset resource="wsc.qrc">
         <normaloff>:/icons/fileopen.png</normaloff>:/icons/fileopen.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Paste</string>
       </property>
       <property name="icon">
        <iconset resource="wsc.qrc">
         <normaloff>:/icons/editpaste.png</normaloff>:/icons/editpaste.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="text">
        <string>Help</string>
       </property>
       <property name="icon">
        <iconset resource="wsc.qrc">
         <normaloff>:/icons/contents.png</normaloff>:/icons/contents.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="sourceTextEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Max &amp;no. of words:</string>
       </property>
       <property name="buddy">
        <cstring>maxWords</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="maxWords">
       <property name="specialValueText">
        <string> No Limit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Min size:</string>
       </property>
       <property name="buddy">
        <cstring>minSize</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="minSize">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Max size:</string>
       </property>
       <property name="buddy">
        <cstring>maxSize</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="maxSize">
       <property name="specialValueText">
        <string> No Limit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="genlistButton">
       <property name="text">
        <string>Generate List</string>
       </property>
       <property name="icon">
        <iconset resource="wsc.qrc">
         <normaloff>:/icons/build.png</normaloff>:/icons/build.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="listTextEdit">
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Open a file or paste some text into the top box then press 'Generate List'.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="wsc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>sourceTextEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>listcreator</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
